### PR TITLE
Added Sql admin field #101

### PR DESCRIPTION
**Pull request by RyanPybus**
- added adminStatus Integer to database sql, 1 if admin, 0 else
- added validateAdmin function to DBUser to check if user is an admin
- added admin field to modifyUser so a user's admin status may be changed
- modified tests to include admin
- also updated comments on method headers

### CODE REVIEW

File: SQLite/DBManager.py [hunk 1]
```diff
@@ -33,27 +49,44 @@ def deleteUser(self, email):
...
         except Exception as e:
-            print("transaction failed: ");
+            print("Operation failed: ");
             print(e);
 
-    # modify user's email/password based on their userid, which must first be retrieved.
-    # if only one field needs modifying, just pass the other as it was retrieved from the database  
-    def modifyUser(self, id, newEmail, newPassword):
+
+    def modifyUser(self, id, newEmail, newPassword, adminstatus):
```
Comment (anthonyargatoff) [inline SQLite/DBManager.py:56]: Great comments!

**Review by anthonyargatoff — APPROVED** [review]
Good looks great. It is very easy to read and contains excellent comments.

**Review by DyPickard — APPROVED** [review]
the changes look good.

**Review by Aayushchaudhary84 — APPROVED** [review]
Looks good

**Pull request merged**